### DISCUSSION TITLE
LabeledExpression and patterns using RegExpr  or * to specify any character

### DISCUSSION THREAD

**Discussion by Ivan23info** [detekt/detekt#4097] [Q&A] [answered]
Hi! I tried a few combinations and it seems that impossible to use RegExpr or short annotations, for example '*' or '**' to specify any character for `LabeledExpression`?
I would like to add `return@` to LabeledExpression, to ignore for example `return@funA 12` and `return@here list`.
If I'm wrong, please, provide example how to do this.

**Comment by cortinico** [comment, marked as answer]
There is no full regex support in that configuration:
https://github.com/detekt/detekt/blob/c35689785869358d38aedce74bf453beafcd3111/detekt-rules-complexity/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/complexity/LabeledExpression.kt#L71

You can specify a string that will be used and matched with a `contains`:

https://github.com/detekt/detekt/blob/c35689785869358d38aedce74bf453beafcd3111/detekt-rules-complexity/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/complexity/LabeledExpression.kt#L80-L82

So `funA`, `here` would work in your case.
